src/tests/ftest/util/test_utils_container.py

```diff
@@ -1084,6 +1031,24 @@ def query(self, *args, **kwargs):
         return self.daos.container_query(
             pool=self.pool.identifier, cont=self.identifier, *args, **kwargs)
 
+    def verify_query(self, expected_response):
+        """Verify daos container query returns expected response values.
+
+        Args:
+            expected_response (dict): expected response values
+
+        Returns:
+            bool: whether response values from daos container query match expected values
+
+        """
+        response = self.query()['response']
```

Comment:
We should think about having `DaosCommand` use its own verison of `_get_json_result()` that verifies the dictionary contains the `response` (and possibly others, `errors` comes to mind), so that in the case where its missing we get a more recognisable `CommandFailure` exception as opposed to a more generic `KeyError` exception.  This is not specific to this PR though.

Reply:
Agree